picard/tagger.py

```diff
@@ -181,6 +181,8 @@ def plugin_dirs():
 
 class ParseItemsToLoad:
 
+    WINDOWS_DRIVE_TEST = re.compile(r"^[a-z]\:", re.IGNORECASE)
```

Comment:
I'm not a Windows expert, but I don't think drive names can be a lowercase letter, nor more than one letter.

If this statement is correct, we can go with:

```suggestion
    WINDOWS_DRIVE_TEST = re.compile(r"^[A-Z]\:")
```

Reply:
Well, ignore, I forgot Win paths are case-insensitive, so `c\:` might be possible.

Reply:
Yeah, I can use lower case drive letters, such as "c:\Ripped Files\Mountain Goats" to access the directcory, and Picard loads the information properly.

Reply:
Any preference on whether to use `re.compile(r"^[a-zA-Z]\:")` or `re.compile(r"^[a-z]\:", re.IGNORECASE)`?